content-src/components/TopSites/TopSiteForm.jsx

```diff
@@ -1,4 +1,5 @@
 import {actionCreators as ac, actionTypes as at} from "common/Actions.jsm";
```

Comment:
Ideally this form should call `onPreviewButtonClick` or `onDoneButtonClick` on a `onSubmit`, which would require some refactoring